packages/cli/test/integration/generators/relation.has-many.integration.js

```diff
@@ -812,6 +813,29 @@ describe('lb4 relation HasMany', function() {
           assert.fileContent(sourceRepositoryFile, regex);
         });
       });
+
+      it('registers the inclusion resolver for the relation', async () => {
+        const singleWordClassConstractor = [
```

Comment:
nitpick: `Constractor` --> `Constructor`?
or you can just use `Ctor` for short :)

Reply:
I copied the name from other tests. I will fix them as well.